src/app/core/services/travelperk/travelperk.service.ts

```
@@ -18,7 +20,8 @@
 
   constructor(
     private apiService: ApiService,
-    private orgService: OrgService
+    private orgService: OrgService,
+    private helperService: HelperService,
```

ESLint

[error] Unexpected trailing comma.

(comma-dangle)